run_examples/run_ogusa_example.py

```diff
@@ -18,9 +19,21 @@
 
 def main():
     # Define parameters to use for multiprocessing
+    RAM_stats = psutil.virtual_memory()
+    RAM_total_bytes = RAM_stats.total
+    RAM_total_GB = RAM_total_bytes / 1073741824
+    mem_per_wkr_txf = 3.5  # Memory per worker (GB) in tax function estimation
```

Comment:
We may want to make put these amounts for the memory for tax function and memory for model solution in `constants.py` so they only need to be adjusted in one place.

Reply:
@jdebacker Agreed. Plus, I just guessed at the `mem_per_wkr_mod=0.05` amount. But I know that it is much less than a gigabyte so that the processor constraint rarely binds. I'll go through and profile the memory footprint before I remove the WIP designation of this PR.